docs/diagrams/StudyManagerSequenceDiagram.puml

```diff
@@ -0,0 +1,36 @@
+@startuml
+!include Style.puml
+
+box LOGIC_COLOR_T1
+participant "studyManager:StudyManager" as studyManager LOGIC_COLOR
+participant "module:Module" as module LOGIC_COLOR
+participant  "ModuleCalendarReader" as icsReader LOGIC_COLOR
+end box
+
+
+[-> studyManager : studyManagerRunner()
+activate studyManager
+alt add command
+    studyManager->studyManager:addModule()
+    activate studyManager
+
+    studyManager -> module ** : new Module(moduleCode, catgeory, day, timeSlot)
+    activate module
+    return :Module
+    deactivate module
+    deactivate studyManager
+
+else read ics command
+    studyManager->studyManager ++: openIcsFile(fileName)
+    studyManager -> icsReader: readIcsFile(fileName)
+    activate icsReader
+    icsReader -> icsReader ++: parseIcsCalendar(fileName)
+    icsReader-> module **: new Module(moduleCode,category,day,timeslot)
```

Comment:
Good use of constructors! Consider shortening the description to something like "create new Module" such that the diagram is at a higher level/more readable?

Reply:
Sounds good! I will look into it!